Source/JavaScriptCore/jit/GCAwareJITStubRoutine.cpp

```diff
@@ -163,6 +163,15 @@ void PolymorphicAccessJITStubRoutine::addedToSharedJITStubSet()
     m_isInSharedJITStubSet = true;
 }
 
+bool PolymorphicAccessJITStubRoutine::visitWeakImpl(VM& vm)
```

Comment:
Who calls this? Is this dead?

Reply:
This is called through `JITStubRoutine::visitWeak` (e.g. InlineCacheHandler's `m_stubRoutine->visitWeak`)

Reply:
I see, this is called via a `derived` callback so it called the inherited one previously